modal.js

```diff
@@ -0,0 +1,24 @@
+const containerModal = document.querySelector(".modal");
+const btnModalOpen = document.querySelector(".btn-modal-open");
+const btnModalClose = document.querySelector(".btn-modal-close");
+
+containerModal.style.display = "none";
+
+const openModelHandler = () => {
+    containerModal.style.display = "flex";
+    btnModalOpen.style.display = "none";
+    console.log(true)
```

Comment:
You can remove console.log from the file. Everything else looks good.

Reply:
yeah, i forgot to remove this